src/settings.ts

```diff
@@ -251,21 +251,22 @@ export async function getEffectiveConfigurationTarget(settingName: string): Prom
 
     const detail = configuration.inspect(settingName);
     let configurationTarget = null;
-    if (typeof detail.workspaceFolderValue !== undefined) {
+    if (typeof detail.workspaceFolderValue !== "undefined") {
```

Comment:
should this be this?
```suggestion
    if (detail.workspaceFolderValue !== undefined) {
```

Reply:
No, `Codacy`/`ESLint` will complain because of not using `typeof` and not using double quotes around `undefined`:
https://eslint.org/docs/rules/no-undefined
Unfortunately, `Codacy` seems to lose its history but those were the complaints that made the build red in my previous commits.

Reply:
I read the link. I think that advice is fine.